src/Microsoft.Health.Fhir.Core/Features/Operations/Export/CreateExportRequestHandler.cs

```diff
@@ -89,7 +95,8 @@ public async Task<CreateExportResponse> Handle(CreateExportRequest request, Canc
                 _exportJobConfiguration.MaximumNumberOfResourcesPerQuery,
                 _exportJobConfiguration.NumberOfPagesPerCommit,
                 request.ContainerName,
-                request.Parallel);
+                request.Parallel,
+                smart: _contextAccessor?.RequestContext?.AccessControlContext?.ApplyFineGrainedAccessControl == true);
```

Comment:
Is SMART the default behavior? 
I had the impression that in the worst case, if everything is null, then smart would be **false**.

Reply:
smart = false is the default behavior.  We only SMART true if everything resolves (not null) and ApplyFineGrainedAccessControl is set to true.

Reply:
Looking at [ExportJobRecord.cs](https://github.com/microsoft/fhir-server/pull/2929/files#diff-d6a0af71536d52566fe4cdfa7742236b4190b705b53afce268d9f9aebaa271e8), I see that the default value of smart is false. Which is what I imagine is the default behavior.